### PR TITLE
Dynamic value for ClearArt on video Pause Screen

### PR DESCRIPTION
**Pull request by KJ-Cox**
Utils.cs adds active window id for video pause for MovingPictures and TVSeries.

FanartSelectedOther.cs prevent TVSeries value from staying visible on pause screen when going to TVSeries then MovingPictures and pausing video resulting in ClearArt logo from TVSeries being visible as well as the current movie.

### CODE REVIEW

File: FanartHandler.sln
Comment (andrewjswan) [inline FanartHandler.sln:1]: Can you remove changes to configuration files (`FanartHandler.sln` and `FanartHandler/FanartHandler.csproj`) from Pull Request? It looks like upgrading and updating them affects the project build in VS 2019.

Comment (KJ-Cox) [reply FanartHandler.sln:1]: Changed as requested. Sorry for any issues this caused. Good to hear you are still OK. Have not seen you around on the forum and I seen on the news there was an attack near Kiev.

Comment (andrewjswan) [reply FanartHandler.sln:1]: > Changed as requested. Sorry for any issues this caused. Good to hear you are still OK. Have not seen you around on the forum and I seen on the news there was an attack near Kiev.

Now there are only project files that needed to be removed, there are no files with changes.

Comment (KJ-Cox) [reply FanartHandler.sln:1]: Files changed 2

**Pull request merged**